tests/test_compressors/model_compressors/test_model_compressor_distributed.py

```
@@ -0,0 +1,341 @@
+from compressed_tensors import ModelCompressor
+from compressed_tensors.quantization import (
+    QuantizationArgs,
+    QuantizationConfig,
+    QuantizationScheme,
+    QuantizationStatus,
+)
+from tests.test_offload.conftest import torchrun
+from tests.testing_utils import requires_gpu
+
+
+def create_quantization_config(bits=4, format="pack-quantized"):
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)